R/compat-dplyr.R

```diff
@@ -79,10 +79,3 @@ last <- function(x) {
 slice <- function(.data, ...) {
   .data[c(...), , drop = FALSE]
 }
-
-#' @importFrom purrr as_mapper map
-map_dfr <- function(.x, .f, ..., .id = NULL) {
-  .f <- as_mapper(.f, ...)
-  res <- map(.x, .f, ...)
-  bind_rows(res, .id = .id)
-}
```

Comment:
We can use `purrr:: map_dfr()` now.

Reply:
Does it not require dplyr anymore? That was the reason why we re-implemented it initially.

Reply:
https://purrr.tidyverse.org/reference/map.html
Ie See. #324.

Reply:
Reverted. I have also added a comment to explain why we are re-implementing this.